_component_design/breadcrumbs.md

```diff
@@ -51,65 +79,126 @@ Breadcrumbs offer a way to articulate the platform architecture in a way the use
+<embed src="{{site.baseurl}}/assets/images/components/navigation/breadcrumbs/breadcrumbs-composition.png" width="318"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Specifications
+### Style
 
-- __Placement:__ Keep breadcrumbs above the major heading of the page, 8px from the eyebrow. The container for breadcrumbs will always live on the top left corner of the page when it is present. Make sure the breadcrumb container is 16px above the content on the top of the page.
-- __Container & Spacing:__ The breadcrumbs container is 28px tall and full-page wide if it needs to be. We recommend shortening measures below. Breadcrumbs are in all caps with arrow icons in `gray 600` in-between.
-- __Alignment:__ Breadcrumbs should be left aligned to the left-most content on the page as noted in [Layout]({{site.baseurl}}/layout/page-layouts.html).
+Use these specifications when you construct breadcrumbs.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [caption:"Detailed specifications for breadcrumbs."] [class:"image bg-light border"] %}
-<embed src="{{site.baseurl}}/assets/images/components/navigation/bread-crumbs/standard-bc.svg"/>
+
+{% figure [caption:"Breadcrumb style"] [class:"image bg-light border"] %}
+
+<embed src="{{site.baseurl}}/assets/images/components/navigation/breadcrumbs/breadcrumbs-style.png" width="386"/>
+
 {% endfigure %}
+
 {% endcolumn %}
+
 </div>
 
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Naming and Abbreviations
+### Spacing
 
-- Ensure the items in the breadcrumbs have the same name as the page it navigates to. If the page title is longer than necessary, use shortened variations of the title. For example, instead of `Billing overview`, use `Overview`.
+Place breadcrumbs above the page header, 8px from the eyebrow. Add an 8px space between a breadcrumb item and the angle-right icon.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [caption:"As depicted, limit your breadcrumb item name."] [class:"image bg-light border"] %}
-<embed src="{{site.baseurl}}/assets/images/components/navigation/bread-crumbs/name-abbrev.svg"/>
+
+{% figure [caption:"Breadcrumb spacing"] [class:"image bg-light border"] %}
+
+<embed src="{{site.baseurl}}/assets/images/components/navigation/breadcrumbs/breadcrumbs-spacing.png" width="641"/>
+
 {% endfigure %}
+
 {% endcolumn %}
+
 </div>
 
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Too many items
+## States
```

Comment:
need to add correct hex code for the current page

Reply:
Done.